### PR TITLE
[Update 8] - Update GraalVM Learn page with Update 8 changes

### PR DESCRIPTION
**Pull request by TharmiganK**
Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/4757

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md [hunk 1]
```diff
@@ -49,56 +49,54 @@ After the environment is set up, follow the steps below to build the native exec
            user/hello_docker:0.1.0
    
    Generating artifacts
-   
+
```
Comment (TharmiganK) [inline swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md:52]:
```suggestion
```

**Review by shafreenAnfar — APPROVED** [review]

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ intro: This guide walks you through compiling a Ballerina application to a nativ
 ## Prerequisites
 
 To complete this part of the guide, you need:
-1. [Ballerina 2201.7.0 (Swan Lake)](/downloads) or greater
+1. [Latest Ballerina Swan Lake distribution](/downloads)
```
Comment (praneesha) [inline swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md:13]:
```suggestion
1. Latest [Ballerina Swan Lake] distribution(/downloads)
```

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md [hunk 3]
```diff
@@ -10,25 +10,23 @@ intro: This guide walks you through compiling a Ballerina application to a nativ
 ## Prerequisites
 
 To complete this part of the guide, you need:
-1. [Ballerina 2201.7.0 (Swan Lake)](/downloads) or greater
+1. [Latest Ballerina Swan Lake distribution](/downloads)
```
Comment (praneesha) [inline swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md:13]:
```suggestion
1. [Latest Ballerina Swan Lake distribution](/downloads)
```
Ditto

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md [hunk 4]
```diff
@@ -10,25 +10,23 @@ intro: This guide walks you through compiling a Ballerina application to a nativ
...
 2. A text editor
    >**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the  <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina extension</a> installed.
 3. GraalVM installed and configured appropriately
 4. A command terminal
 
 ## Configure GraalVM
 
-1. Install GraalVM on Linux, macOS, and Windows (via Git Bash, Cygwin, or WSL) using the following command.
+1. Install GraalVM using [SDKMAN!](https://sdkman.io/). For additional download options, see [Get Started with GraalVM](https://www.graalvm.org/latest/docs/getting-started/).
```
Comment (praneesha) [inline swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md:21]:
```suggestion
1. Install GraalVM using [SDKMAN!](https://sdkman.io/). For additional download options, see [Get Started with GraalVM](https://www.graalvm.org/latest/docs/getting-started/).
```
Make the second sentence a tip.

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md [hunk 5]
```diff
@@ -10,25 +10,23 @@ intro: This guide walks you through compiling a Ballerina application to a nativ
...
-      For additional information, see [Get Started with GraalVM](https://www.graalvm.org/22.3/docs/getting-started).
-2. Configure the runtime environment. Set the `GRAALVM_HOME` environment variable to the GraalVM installation directory as directed at the end of the execution of the above command.
+      > **Note:** If you have installed Ballerina Swan Lake Update 7(2201.7.x) or lower, you have to install GraalVM JDK 11. For download options, see [Get Started with GraalVM](https://www.graalvm.org/22.3/docs/getting-started/macos/).
 
 > **Note:** 
-> - On Windows, the native image requires Visual Studio Code and Microsoft Visual C++ (MSVC). For instructions on installing Visual Studio Code with the Windows 10 SDK, go to [Using GraalVM and Native Image on Windows 10](https://medium.com/graalvm/using-graalvm-and-native-image-on-windows-10-9954dc071311).
+> - On Windows, the native image requires Visual Studio Code and Microsoft Visual C++ (MSVC). For more details, see [Prerequisites for Native Image on Windows](https://www.graalvm.org/latest/docs/getting-started/windows/#prerequisites-for-native-image-on-windows).
```
Comment (praneesha) [inline swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md:29]:
```suggestion
> - On Windows, the native image requires Visual Studio Code and Microsoft Visual C++ (MSVC). For more details, see [Prerequisites for Native Image on Windows](https://www.graalvm.org/latest/docs/getting-started/windows/#prerequisites-for-native-image-on-windows).
```
Make the second sentence an info section.

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md (hunk 2 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-in-a-container.md:13]:
```suggestion
1. Latest [Ballerina Swan Lake](/downloads) distribution
```

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md (hunk 3 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md:13]:
```suggestion
1. Latest [Ballerina Swan Lake](/downloads) distribution
```

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md (hunk 5 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md:29]: This is already inside a Note

File: swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md (hunk 4 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md]: Fixed via [2d302dd](https://github.com/ballerina-platform/ballerina-dev-website/pull/7676/commits/2d302ddb7f82fd79ff8579a97fd7140604c41df3)

**Review by praneesha — APPROVED** [review]

**Pull request merged**